packages/dialtone-vue2/components/tooltip/tooltip.vue

```diff
@@ -12,25 +12,19 @@
       @mouseenter="onEnterAnchor"
       @mouseleave="onLeaveAnchor"
       @keydown.esc="onLeaveAnchor"
-      @touchstart="onTouchStart"
     >
       <!-- @slot Slot for the anchor element -->
       <slot
         name="anchor"
       />
     </span>
-    <dt-lazy-show
+    <div
       :id="id"
       ref="content"
-      :show="isVisible"
-      role="tooltip"
-      aria-hidden="false"
       data-qa="dt-tooltip"
-      appear
-      :transition="transition"
```

Comment:
`transition` prop is not being used anymore, seems like `fade` is the default from tippy.

Reply:
nevermind, I think there is still some value in being able to set transition to false for no transition, so I will make it work like this.